src/common/POLFields/EresourcesFields/FieldURL.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { Field } from 'react-final-form';
+
+import { TextField } from '@folio/stripes/components';
+import { validateURL } from '@folio/stripes-acq-components';
+
+const FieldURL = ({ disabled, required }) => {
```

Comment:
it makes no sense to add prop by prop, because you have to manually add them every time. To make component common and customizable you need an ability to rewrite existing props